main.go

```diff
@@ -37,7 +38,11 @@ func init() {
 		panic(err)
 	}
 
-	logger = log.New(os.Stdout, "", log.LstdFlags)
+	logger = logrus.New()
+
+	if os.Getenv("CALICO_DEBUG") != "" {
+		logrus.SetLevel(logrus.DebugLevel)
```

Comment:
You also need to change the log level for the libcalico-go library I think

See https://github.com/projectcalico/calico-cni/blob/master/utils/utils.go#L214